lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -141,4 +149,39 @@ sub press_ok_clone {
     send_key($self->{ok_clone_shortcut});
 }
 
+sub modify_create_new_partition_table {
+    my ($self, $item) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    _select_system_view_section();
+    send_key_until_needlematch(SELECTED_HARD_DISKS, 'down');
+    send_key($self->{modify_hard_disk_shortcut});
+    send_key_until_needlematch(SELECTED_CREATE_NEW_TABLE, 'down');
```

Comment:
We can exclude last two lines in a new method `create_new_partition_table`.

Reply:
Ah, ok. Now the "Modify_Hard_Disk" name would be clear.